### PR TITLE
Fixes #7742: rudder CLI commands should print their expected options with -h / --help

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/7742

### CODE REVIEW

**Comment by amousset** [comment]
PR updated

**Comment by amousset** [comment]
Displays:
help

**Comment by amousset** [comment]
PR updated

**Comment by amousset** [comment]
PR updated

**Comment by jooooooon** [comment]
Ahhh ! I just reviewed this, before your updated the PR @amousset. What changed?

File: bin/rudder [hunk 1]
```diff
@@ -104,6 +116,15 @@ then
...
 fi
 
+# test if third parameter is --help
+option=`echo "$1" | sed 's/[^A-Za-z_-]*//g'`
+if [ "${option}" = "--help" ] || [ "${option}" = "-h" ] || [ "${option}" = "help" ]
+then
+  options=`sed -ne "/#[ \t]*@man[ \t]*/s/#[ \t]*@man[ \t]*//p" "${BASEDIR}/${role}-${command}" |
+           sed "s/*//g" | sed "s/+//g" | sed 's/^-/  -/' | sed 's/^/  /'`
+  command_help "${role}" "${command}" "${options}"
+  exit 3
```
Comment (jooooooon) [inline bin/rudder]: Are you sure we should exit with an error code, when we did what was asked?

**Comment by amousset** [comment]
I added this line: https://github.com/Normation/rudder-agent/pull/41/files#diff-9d0f8e5d6046dde44f79a7362121364aR44.

**Comment by amousset** [comment]
PR updated

File: bin/rudder (hunk 1 above)
Comment (amousset) [inline bin/rudder]: Clearly not, fixed.

**Comment by jooooooon** [comment]
Great stuff! Thanks.

**Pull request merged**